### DISCUSSION TITLE
Livewire v3 support

### DISCUSSION THREAD

**Discussion by PhiloNL** [wire-elements/modal#329] [Announcements]
Hi everyone!

Livewire v3 beta has arrived, and there are many amazing new features. 
You are probably wondering when you can use the modal component with v3, and at this time, ~~I don't have an estimate just yet~~. I am in the progress of making the required changes, but given that v3 is still in beta, I'm running into various bugs and creating PR's to get those resolved. But be assured; it's being worked on.

**Update**
The first beta release of the modal component is available:
https://github.com/wire-elements/modal/releases

Oh by the way, I did a quick video about a few of those in case you want to learn more about v3: 
https://www.youtube.com/watch?v=0r6XX0_O7Kc

[![Livewire v3](https://img.youtube.com/vi/0r6XX0_O7Kc/0.jpg)](https://www.youtube.com/watch?v=0r6XX0_O7Kc)

**Comment by vlados** [comment]
any way we can help?

**Reply by PhiloNL** [reply]
Thanks for the offer!
Unfortunately, not at this time, I'm waiting for PRs to be merged and a few bugs to be resolved before I can continue the upgrade process.

**Reply by ev3rlost** [reply]
Hey Philo! Loved the YT video! You should definitely do more of those for Livewire V3 (new ways of doing data tables, your modals, drawers ... the stuff we've been waiting for)! Any insights of when the stable release is due? I've been tinkering with V3 for the last two days and found it's very wonky for the most part and the docs are missing loads of things what Caleb showed/demoed at Laracon (btw, the YT Laracon video went private, so there's that, lol). I'm realy eager to build an app with Livewire V3 and your updated Modal package (which is phenomenal btw!) - having been waiting since Sept. '22. Thanks and cheers!

**Reply by PhiloNL** [reply]
Thanks @ev3rlost :) Appreciate the kind words. I'll definitely do more YT videos. I think the Laracon videos should be up somewhere next week.

Given the Livewire v3 beta is only available for a few days, the stable release will probably depend on the issues being reported (I guess). You would have to reach out to Caleb for more info. 

Happy to hear you've been using the modal package! Thanks again for the kind words 😄

**Reply by jeffchown** [reply]
Thanks, @PhiloNL!  Looking forward to using your package with Livewire 3!

**Comment by PhiloNL** [comment]
A beta release is now available https://github.com/wire-elements/modal/releases
